src/js/tech/tech.js

```diff
@@ -989,6 +989,14 @@ Tech.prototype.featuresPlaybackRate = false;
  */
 Tech.prototype.featuresProgressEvents = false;
 
+/**
+ * Boolean indicating wether the `Tech` supports the `sourceset` event.
+ *
+ * @type {boolean}
+ * @default
+ */
+Tech.prototype.featuresSourceset = false;
```

Comment:
so by default, techs will not trigger `sourceset` but a particular tech can override it and trigger a `sourceset`?

Reply:
I would think so, since its not really part of the spec, but I could see it being really helpful to have all techs do it, maybe in a major version bump?

Reply:
Most techs that aren't html5 can't get a source set directly like you can with the video element, so, might be ok to just trigger it in the `tech`'s `setSource()` method? But I think this is fine as a way forward for other techs.

Reply:
Yeah Just triggering in there may make sense, and setting this to true also makes sense